components/cli/pkg/util/aws.go

```diff
@@ -59,7 +61,20 @@ func DownloadFromS3Bucket(bucket, item, path string, displayProgressBar bool) {
 	writer.display = displayProgressBar
 
 	writer.init(s3ObjectSize)
-	_, err = downloader.Download(writer,
+	if displayProgressBar {
+		go func() {
+			var size int64 = 0
+			for size < s3ObjectSize {
+				size, err = GetFileSize(filepath.Join(path, item))
+				if err != nil {
+					log.Fatal(err)
```

Comment:
log.Fatal will kill the running process. Can't we just log the error and continue? Otherwise, download can be stopped in its final stage just trying to show the progress bar.

Reply:
Updated.